inputfiles/idlSources.json

```diff
@@ -438,6 +438,10 @@
         "url": "https://www.w3.org/TR/referrer-policy/",
         "title": "Referrer Policy"
     },
+    {
+        "url": "https://www.w3.org/TR/resize-observer/",
```

Comment:
https://drafts.csswg.org/resize-observer/ is a better choice if the old version is faulty. Manually fixing IDL will require same fix every time anyone call `npm run fetch-idl`, which is not we want.

Reply:
BTW I also filed an issue to remove `ResizeObservation` which is not supposed to be here: https://github.com/w3c/csswg-drafts/issues/5660

Reply:
> drafts.csswg.org/resize-observer is a better choice if the old version is faulty. Manually fixing IDL will require same fix every time anyone call npm run fetch-idl, which is not we want.

Ah, sure! Didn't know about that, thanks!

> BTW I also filed an issue to remove ResizeObservation which is not supposed to be here: w3c/csswg-drafts#5660

Should I leave it here until it will be removed from IDL, right?

Reply:
You can remove it by add a name in `removedTypes.json`. It makes sure we ignore the type whenever we update the IDL file 👍

Reply:
Done in a85006d and 1c32701.

Thank you so much @saschanaz!